yt/fields/fluid_fields.py

```diff
@@ -56,6 +57,31 @@ def _cell_mass(field, data):
     )
     registry.alias((ftype, "mass"), (ftype, "cell_mass"))
 
+    # momentum
+    def momentum_xyz(v):
+        def _mom(field, data):
+            return data["gas", "mass"] * data["gas", f"velocity_{v}"]
+
+        def _momd(field, data):
+            return data["gas", "density"] * data["gas", f"velocity_{v}"]
+
+        return _mom, _momd
+
+    for v in "xyz":
```

Comment:
careful, these only make sense in cartesian coordinate systems. I believe the more general implementation involves
```python
for ax in registry.ds.coordinates.axis_order:
    ...
```